sqlglot/dialects/databricks.py

```diff
@@ -0,0 +1,39 @@
+from sqlglot import exp
+from sqlglot.dialects.spark import Spark
+from sqlglot.helper import list_get
+
+
+def generate_date_delta(self, e):
```

Comment:
can you dry this up with the tsql stuff? put it into dialect.py or a shared module

Reply:
It is really only applicable to TSQL, Databricks. Would putting it in Dialect not be setting false expectations?

Reply:
you can import from tsql